src/azure-cli/HISTORY.rst

```diff
@@ -21,6 +21,12 @@ Release History
 * [BREAKING] When not specified, the default value for `--start-task-wait-for-success` on `az batch pool create` is now true (was false).
 * [BREAKING] The default value for Scope on AutoUserSpecification is now always Pool (was Task on Windows nodes, Pool on Linux nodes). This argument is not exposed via the commandline, but can be set in the `--json-file` arguments.
 
+**Compute**
```

Comment:
What is the output of the vmss show command?
Is it the json output of the response?

Reply:
> 
> 
> What is the output of the vmss show command?
> Is it the json output of the response?


`vmss show` visit `GET https://management.azure.com/subscriptions/{...}/resourceGroups/{...}/providers/Microsoft.Compute/virtualMachineScaleSets/{name}?api-version=2019-03-01`.

Basically, Yes.

Reply:
And, `vmss get-instance-view` visit `GET https://management.azure.com/subscriptions/{...}/resourceGroups/{...}/providers/Microsoft.Compute/virtualMachineScaleSets/{name}/instanceView?api-version=2019-03-01`

Reply:
What's the difference between `virtualMachineScaleSets/{name}?$expand=instanceView` and `virtualMachineScaleSets/{name}/instanceView`?

Reply:
virtualMachineScaleSets/{name}?$expand=instanceView returns the vmScaleSet goal state with a property named instance view populated (instanceView is expensive runtime state)

virtualMachineScaleSets/{name}/instanceView just returns the expensive runtimestate without the vmScaleSet goal state infromation